src/main/java/uk/nhs/digital/ps/migrator/config/ExecutionConfigurator.java

```diff
@@ -166,7 +170,12 @@ private Path getPathArg(final ApplicationArguments args, final String commandLin
                 "Path to field mapping spreadsheet used to map migrated datasets' P code to " +
                     "Coverage Start, Coverage End, Geographic Coverage, Granularity." +
                     " Required if --" + NESSTAR_CONVERT_FLAG + " is specified."
-            )
+            ),
+            describe(
+                NATIONAL_INDICATORS_IMPORT_PATH,
+                "Path to the spreadsheet that contains the national indicators we want to import into hippo." +
+                    " Required for indicator imports, optional otherwise."
```

Comment:
Would you mind adding something along the lines of '_NOT to be used together with arguments that trigger other conversions (e.g. Nesstar); this is to prevent the tasks inadvertently deleting each other's output as they drop and recreate the exim-import directory_'?

This deletion and re-creation of the exim-import folder (the one where the JSON files are being generated to) is something I added to your task after it's failed for me when it couldn't find the directory; I neglected to update this description, though and just noticed my omission. We could introduce separate output directories but I think there is little value in that as it's unlikely that we'd ever need to run both conversions at the same time.

Reply:
Agreed